Examples/NSXBuildFromScratch.ps1

```diff
@@ -580,11 +593,22 @@ $InternalESG_IpSet = New-NsxIPSet -name InternalESG_IpSet -IPAddresses $EdgeInte
 write-host -foregroundcolor "Green" "Creating Security Groups"
 
 #Create SecurityGroups and with static includes
-$WebSg = New-NsxSecurityGroup -name $WebSgName -description $WebSgDescription -includemember (get-vm | ? {$_.name -match "Web0"})
-$AppSg = New-NsxSecurityGroup -name $AppSgName -description $AppSgDescription -includemember (get-vm | ? {$_.name -match "App0"})
-$DbSg = New-NsxSecurityGroup -name $DbSgName -description $DbSgDescription -includemember (get-vm | ? {$_.name -match "Db0"})
+$WebSg = New-NsxSecurityGroup -name $WebSgName -description $WebSgDescription -includemember $WebSt
+$AppSg = New-NsxSecurityGroup -name $AppSgName -description $AppSgDescription -includemember $AppSt
+$DbSg = New-NsxSecurityGroup -name $DbSgName -description $DbSgDescription -includemember $DbSt
 $BooksSg = New-NsxSecurityGroup -name $BooksSgName -description $BooksSgName -includemember $WebSg, $AppSg, $DbSg
 
+# Apply Security Tag to VM's for Security Group membership
+
+$WebVMs = Get-Vm | ? {$_.name -match ("Web0")}
+$AppVMs = Get-Vm | ? {$_.name -match ("App0")}
+$DbVMs = Get-Vm | ? {$_.name -match ("Db0")}
+
+
+Get-NsxSecurityTag $WebStName | New-NsxSecurityTagAssignment -ApplyToVm -VirtualMachine $WebVMs | Out-Null
```

Comment:
rather than get-nsxsecuritytag again - use the $webst/appst/dbst variables defined in lines 582-584